Packs/IronscalesEventCollector/Integrations/IronscalesEventCollector/IronscalesEventCollector.py

```diff
@@ -163,13 +170,10 @@ def get_events_command(
     client: Client,
     args: Dict[str, Any]
 ) -> Tuple[CommandResults, List[Dict[str, Any]]]:
-    events: List[Dict[str, Any]] = []
-
-    for i in client.get_open_incident_ids():
-        incident = client.get_incident(i)
-        events.extend(incident_to_events(incident))
-        if len(events) >= args.get('limit', DEFAULT_LIMIT):
-            break
+    limit: int = arg_to_number(args.get('limit')) or DEFAULT_LIMIT
+    since_time = arg_to_datetime(args.get('since_time') or DEFAULT_FIRST_FETCH, settings=DATEPARSER_SETTINGS)
```

Comment:
Usually, I don't recall we add such an argument but it might be good, nevertheless, we would still want to have the option to use the last run, in cases we want to debug something in the fetch. So maybe when the `since_time` is not given we will use the `last_run`?

Reply:
is it reachable through a non-fetching command? AFAIK not...